packages/database/lib/schema.ts

```diff
@@ -896,6 +896,29 @@ export const proposalTotalTradeVolume = pgView("proposal_total_trade_volume", {
   GROUP BY pass_market.proposal_acct, pass_market_acct, fail_market_acct
   `);
 
+export const userDeposits = pgTable(
+  "user_deposits",
+  {
+    txSig: transaction("tx_sig")
+      .references(() => transactions.txSig)
+      .notNull(),
+      
+    userAcct: pubkey("user_acct")
+      .notNull()
+      .references(() => users.userAcct),
+    
+    tokenAmount: tokenAmount("token_amount").notNull(),
+
+    mintAcct: pubkey("mint_acct")
```

Comment:
Actually I think this should just reference the tokens.mint_acct table. Although it will be the same as this column in the condition vault table, the information in the conditional vault table is not as important for us.